src/Elm/Parser/Tokens.elm

```diff
@@ -218,7 +218,7 @@ typeName =
     Core.variable
         { start = Unicode.isUpper
         , inner = \c -> Unicode.isAlphaNum c || c == '_'
-        , reserved = reservedList
+        , reserved = Set.empty
```

Comment:
Oh good catch!

Reply:
on that note, as you've benchmarked, it might be faster to check for reserved words after `variable` has been parsed using a `case` instead of the current `Set`. Do you think that's a reasonable change?

Reply:
(Did I benchmark that?) Could be reasonable, but I'd imagine a `Set` to be faster. To the benchmarks :smile:

Reply:
I was talking about https://github.com/jfmengels/elm-benchmarks/blob/master/src/WhatIsFaster/SetMemberOrMultipleComparisons-Results-Chrome.png

Reply:
Oh right! Yeah, let's benchmark this with the strings in the reserved list. There are more strings and they're longer so that may have an impact).

Reply:
Result (ran it twice with the same result): 
![image](https://github.com/user-attachments/assets/87aab438-e14b-4a24-b9f2-107892fae7b1)
It's not as much as I'd have hoped but still nice due to how often it's called.

Reply:
something substantial:
![image](https://github.com/user-attachments/assets/451a8f1c-e3ac-4bbe-831c-08a47097b874)
by shortcutting `unicode.isAlphaNum` with `Char.isAlphaNum c`. This should be safe, right?

Reply:
Agreed, good idea